packages/dialog/src/vaadin-dialog.js

```diff
@@ -33,6 +33,10 @@ registerStyles(
       pointer-events: auto;
     }
 
+    ::slotted([slot='title']) {
+      pointer-events: auto;
```

Comment:
Is this needed? The above CSS rule also applies `pointer-events: auto` to `::slotted([slot='title'])`.

Reply:
No, it was added when I was prototyping the solution and was testing not applying `display: contents` to `::slotted([slot='title'])`. It should've been rolled back.